src/carplay.cs

```diff
@@ -1008,4 +1289,556 @@ interface CPTemplateApplicationDashboardScene {
 		[Export ("dashboardWindow", ArgumentSemantic.Strong)]
 		UIWindow DashboardWindow { get; }
 	}
+
+	// warning BI1115: bgen: The parameter 'completionBlock' in the delegate 'CarPlay.CPListItemHandler' does not have a [CCallback] or [BlockCallback] attribute. Defaulting to [CCallback].
+	delegate void CPListItemHandler (CPBaseListItem item, Action completionBlock);
+
+	[Deprecated (PlatformName.iOS, 14, 0)]
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPBaseListItem : NSSecureCoding
+	{
+		[Deprecated (PlatformName.iOS, 14, 0)]
+		[NullAllowed, Export ("listItemHandler", ArgumentSemantic.Copy)]
+		CPListItemHandler ListItemHandler { get; set; }
+
+		[NullAllowed, Export ("userInfo", ArgumentSemantic.Strong)]
+		NSObject UserInfo { get; set; }
+	}
```

Comment:
If it was added and deprecated in iOS 14, why even add it? Our users have never seen it, we can move it to ignore AFAIK.

Reply:
I'm not sure about this one. Looking for CPBaseListItem in the Apple headers shows that it's still used by several API, for example:

```
API_AVAILABLE(ios(12.0)) API_UNAVAILABLE(macos, watchos, tvos)
@interface CPListItem : CPBaseListItem <CPSelectableListItem>
```

We could leave all API using CPBaseListItem out of the binding for now until Apple decides whether they're going to go with CPBaseListItem or CPSelectableListItem.

It seems like the "right" thing to do to avoid exposing an API that's only going to briefly available (and then deprecated) in the betas, but we will be leaving a not insignificant chunk of code unbound of we do this.

@spouliot and others, thoughts?

Reply:
Whenever in (reasonable) doubt don't add the API
It's much easier to add it later than to remove it :)

In this case you could bind it as `@interface CPListItem : NSObject <CPSelectableListItem>`

Reply:
Removed in beta 6! 💯